clippy_lints/src/nonstandard_macro_braces.rs

```diff
@@ -184,6 +184,10 @@ fn macro_braces(conf: FxHashSet<MacroMatcher>) -> FxHashMap<String, (String, Str
             name: "vec",
             braces: ("[", "]"),
         ),
+        macro_matcher!(
+            name: "matches",
+            braces: ("(", ")"),
+        ),
```

Comment:
I think it would be nice to add a test even though the change is trivial so that someone modifying the code latter won't break it

Reply:
This is just default configuration, if there are no tests at all for this feature I can kind of see your point, otherwise not. In any case I unfortunately don't have much time to figure out how to add completely new testing for this feature, but if there are already some tests and you insist of having them extended for this change, let me know where to find them.

Reply:
[The tests are here](https://github.com/rust-lang/rust-clippy/blob/master/tests/ui-toml/nonstandard_macro_braces/conf_nonstandard_macro_braces.rs). If you don't have time to add them I don't think it's a blocker, I can add them later

Reply:
Added a test and squashed everything. The output in the particular test case I've added is a *little* strange since it re-"format"s `{}` to `()` inside the macro invocation. Should I open an issue about that or is it something known from other lint suggestions that already has an issue?

Reply:
I've opened an issue: https://github.com/rust-lang/rust-clippy/issues/9498 Looks like it's there from the initial implementation. Would you like to fix it at the same time? If not it can be done independently

Reply:
I've opened https://github.com/rust-lang/rust-clippy/pull/9499 and rebase on your work so you won't have any conflit